modules/new_profile/php/new_profile.class.inc

```diff
@@ -170,21 +170,17 @@ class New_Profile extends \NDB_Form
         $maxYear   = (isset($endYear, $ageMin)) ? $endYear - $ageMin : null;
 
         // Get sites for the select dropdown
-        $user_list_of_sites = $user->getData('CenterIDs');
-        $num_sites          = count($user_list_of_sites);
-        $psc_labelOptions   = array();
-        if ($num_sites > 1) {
```

Comment:
since you are removing this you will need to modify the JSX file to remove the if site is not null -> display

Reply:
> since you are removing this you will need to modify the JSX file to remove the if site is not null -> display

I believe this part is not necessary. The site could be null if the DSC table has nothing.

Reply:
but then there is no use of hiding it since it's required.